eslzArm/eslzArm.json

```diff
@@ -2749,6 +2795,131 @@
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').denyCorpNetworkingPolicyAssignment]"
+                },
+                "parameters": {
+                    "enforcementMode": {
+                        "value": "Default"
+                    }
+                }
+            }
+        },
+        {
+            // Assigning policy to audit deployment of Private Link / Private DNS Zones to landing zones management group if condition is true
+            "condition": "[equals(parameters('auditCorpNetworking'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "name": "[variables('deploymentNames').auditCorpNetworkingPolicyDeploymentName]",
+            "scope": "[variables('scopes').corpManagementGroup]",
+            "location": "[deployment().location]",
+            "dependsOn": [
+                "policyCompletion"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').auditCorpNetworkingPolicyAssignment]"
+                },
+                "parameters": {
+                    "enforcementMode": {
+                        "value": "Default"
+                    }
+                }
+            }
+        },
+        {
+            // Assigning policy to audit deployment of WAF on Application Gateways to landing zones management group if condition is true
+            "condition": "[equals(parameters('auditCorpNetworking'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "name": "[variables('deploymentNames').auditAppGWWafPolicyDeploymentName]",
+            "scope": "[variables('scopes').lzsManagementGroup]",
+            "location": "[deployment().location]",
+            "dependsOn": [
+                "policyCompletion"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').auditAppGwWafPolicyAssignment]"
+                },
+                "parameters": {
+                    "enforcementMode": {
+                        "value": "Default"
+                    }
+                }
+            }
+        },
+        {
+            // Assigning policy to audit Azure Compute Security Baseline compliance to the landing zones management group if condition is true
+            "condition": "[equals(parameters('enforceACSB'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "name": "[variables('deploymentNames').enforceACSBPolicyDeploymentName]",
+            "scope": "[variables('scopes').lzsManagementGroup]",
```

Comment:
do we think this scope should be higher, like the int root? or also assigned to platform?

Reply:
This was the agreed scope in our review. Perhaps leave at this scope but review in the next cycle?

Reply:
I hear you, but im finding it hard to justify just doing on Landing Zones as this means anyting in platform will be missed, and you're likely care about this more on things like ADDS DCs in the identity subs than apps. 

Can we change this to be scoped to the intermediate root management group please, this follows our approach for ASB/MCSB that is assigned at the same scope.